tests/io/datasets/test_dask_ds.py

```diff
@@ -0,0 +1,217 @@
+import numpy as np
+import pytest
+import dask.array as da
+
+from libertem.udf.sumsigudf import SumSigUDF
+from libertem.udf.sum import SumUDF
+from libertem.io.dataset.base import DataSetException
+from libertem.io.dataset.dask import DaskDataSet
+
+from utils import dataset_correction_verification
+from utils_dask import _mk_dask_from_delayed
+
```

Comment:
Currently this is all tested with an `InlineJobExecutor`. Maybe at least one test can be on a distributed executor, for example like in https://github.com/LiberTEM/LiberTEM/blob/d294d5d7b06dcb3ea6954c804b6eef53dc281fa5/tests/io/datasets/test_frms6.py#L318-L336

The benchmark is already on a distributed context, so I guess it will work just fine. Having one test here makes sure it is run in CI.

Reply:
Ah, I hadn't appreciated that detail of the fixturing. I'll make that change.

Reply:
No worries, I'm already writing one!

Reply:
Thanks!